src/agent/README.md

```diff
@@ -84,6 +84,33 @@ To install the `protoc` command on a Fedora/CentOS/RHEL system:
 $ sudo dnf install -y protobuf-compiler
 ```
 
+## Configuring the agent
+
+The agent can be configured in a couple of ways.
+
+If running under the supervision of the Kata runtime, the agent will be automatically configured according to some properties in the runtime's `configuration.toml` file. For example, the `enable_debug` option under the `[agent.kata]` section can be used to set debug-level logs. However, not all the accepted configurations are mapped into `configuration.toml`, so any additional options should be passed directly to the agent via kernel parameters.
```

Comment:
Would be good to point out how kernel parameters are passed.

Reply:
Agree with @amshinde , it's better to give an example like `kernel_params = "agent.log=debug agent.debug_console agent.debug_console_vport=1026 agent.hotplug_timeout=10"`.

Reply:
Thanks for your comments @amshinde and @liubin ! I agree with your suggestion, I will send a v2 of this PR soon.